cmd/root.go

```diff
@@ -124,8 +127,35 @@ func modifyToSentryLogger(log *zap.Logger, client *sentry.Client) *zap.Logger {
 	}
 
 	log = zapsentry.AttachCoreToLogger(core, log)
-
-	return log.With(zapsentry.NewScope())
+	kernelVersion := "Not linux"
+	if runtime.GOOS == "linux" {
+		cmd := exec.Command("uname", "-r")
+		kernelBytes, err := cmd.Output()
+		if err != nil {
+			log.Debug("failed to get kernel version", zap.Error(err))
+		} else {
+			kernelVersion = string(kernelBytes)
+		}
+	}
+	arch := runtime.GOARCH
+	installationID, err := fs.NewTeleFS(log).Get(false)
```

Comment:
Does telemetry gets initiated before we reach this line..? If it doesn't we will not send installation-id of first time user if that user faces any problem right? Need to change this approach.